packages/clarity-js/src/interaction/submit.ts

```diff
@@ -0,0 +1,30 @@
+import { Event } from "@clarity-types/data";
+import { SubmitState } from "@clarity-types/interaction";
+import { bind } from "@src/core/event";
+import { schedule } from "@src/core/task";
+import { time } from "@src/core/time";
+import encode from "./encode";
+import { target } from "@src/layout/target";
+
+export let state: SubmitState[] = [];
+
+export function start(): void {
+    reset();
+}
+
+export function observe(root: Node): void {
+    bind(root, "submit", recompute, true);
+}
+
+function recompute(evt: UIEvent): void {
+    state.push({ time: time(), event: Event.Submit, data: { target: target(evt), hash: null } });
```

Comment:
There is also a "submitter" property on this event. Worth capturing? 

[https://developer.mozilla.org/en-US/docs/Web/API/SubmitEvent](https://developer.mozilla.org/en-US/docs/Web/API/SubmitEvent)

Based on taht I imagine the target points to the form elmeent? Can you confirm?